packages/backend/server/src/plugins/copilot/providers/fal.ts

```diff
@@ -204,20 +238,20 @@ export class FalProvider
     });
   }
 
-  async generateText(
+  async text(
```

Comment:
The output type of all models is ModelInputType.Image. Can `text` and `streamText` be deleted because they are useless?

Reply:
some selfhosted user using fal's text models

Reply:
Should `ModelOutputType.Text` be added to the output type of the model?

Reply:
they are currently modifying the configuration themselves and are not using a fixed model
let me add a method to allow them add model define with config.json